quipucords/api/deployments_report/model.py

```diff
@@ -115,6 +115,7 @@ class SystemFingerprint(models.Model):
     system_creation_date = models.DateField(null=True)
     system_last_checkin_date = models.DateField(null=True)
 
+    system_purpose = models.TextField(unique=False, null=True)
```

Comment:
you should also add this to the `__str__` function

Reply:
Good point!